### PR TITLE
Update 0x17-V9-Communications.md

### PR DESCRIPTION
**Pull request by csfreak92**
Added 9.4.4 as my suggestion for mitigating the risk mention re: usage of wildcard TLS certs, issue #995. 

@tghosth, your branch of modifications looks good, I cannot think of anything to modify and clean-up. Note: This PR does not include the cert management requirements that @mgargiullo proposes, I'll leave it to you for that since you are more knowledgeable about protection and management of certs, but I kind of mentioned it here a bit.

### CODE REVIEW

File: 5.0/en/0x17-V9-Communications.md [hunk 1]
```diff
@@ -51,13 +51,14 @@ HTTP traffic between internal facing services should also be encrypted, idealy u
 
 ## V9.4 General TLS Security Guidance
 
-Use secure TLS configuration and use up to date tools to review the configuration on a regular basis.
+Use secure TLS configuration and use up to date tools to review the configuration on a regular basis. While usage of wildcard TLS certificates is not inherently secure, but a compromise of a certificate that is deployed across all owned environments (e.g. production, staging, development, test, etc.) may lead to compromise of the security posture of the applications using it. Proper protection, management, and usage of separate TLS certificates in different environments should be employed if possible.
```
Comment (tghosth) [inline 5.0/en/0x17-V9-Communications.md:54]: Do you mean:

>...usage of wildcard TLS certificates is not inherently **in**secure...?

File: 5.0/en/0x17-V9-Communications.md [hunk 2]
```diff
@@ -51,13 +51,14 @@ HTTP traffic between internal facing services should also be encrypted, idealy u
...
 
 | # | Description | L1 | L2 | L3 | CWE |
 | :---: | :--- | :---: | :---: | :---: | :---: |
 | **9.4.1** | [MOVED FROM 9.1.2] Verify using up to date TLS testing tools that only strong cipher suites are enabled, with the strongest cipher suites set as preferred. | ✓ | ✓ | ✓ | 326 |
 | **9.4.2** | [MOVED FROM 9.1.3] Verify that only the latest recommended versions of the TLS protocol are enabled, such as TLS 1.2 and TLS 1.3. The latest version of the TLS protocol should be the preferred option. | ✓ | ✓ | ✓ | 326 |
 | **9.4.3** | [MOVED FROM 9.2.4] Verify that proper certification revocation, such as Online Certificate Status Protocol (OCSP) Stapling, is enabled and configured. | | ✓ | ✓ | 299 |
```
Comment (tghosth) [inline 5.0/en/0x17-V9-Communications.md:60]: Are wildcard certificates definitely not mentioned anywhere else in the standard?

File: 5.0/en/0x17-V9-Communications.md [hunk 3]
```diff
@@ -51,13 +51,14 @@ HTTP traffic between internal facing services should also be encrypted, idealy u
...
+| **9.4.4** | [ADDED] Verify that TLS wildcard certificates are not used across all deployed environments. If using TLS wildcard certificates cannot be prevented, ensure that unique certificates are deployed in the production environment and non-production environments. | ✓ | ✓ | ✓ | 1008 |
```
Comment (tghosth) [inline 5.0/en/0x17-V9-Communications.md:61]: What do you think about the following:

> Verify that if TLS wildcard certificates are used, wildcard certificates from a non-production environment are not valid for a production environment.

**Review by tghosth — CHANGES_REQUESTED** [review]
hi @csfreak92, please review my comments

File: 5.0/en/0x17-V9-Communications.md (hunk 2 above)
Comment (csfreak92) [reply 5.0/en/0x17-V9-Communications.md:60]: Yes, there were no mentioning of wildcard certificates anywhere in the standard. :)

File: 5.0/en/0x17-V9-Communications.md (hunk 3 above)
Comment (csfreak92) [reply 5.0/en/0x17-V9-Communications.md:61]: Oooh, I like that better @tghosth!

File: 5.0/en/0x17-V9-Communications.md (hunk 1 above)
Comment (csfreak92) [reply 5.0/en/0x17-V9-Communications.md:54]: Woops, yup that's what I meant.

File: 5.0/en/0x17-V9-Communications.md (hunk 3 above)
Comment (tghosth) [reply 5.0/en/0x17-V9-Communications.md:61]: Please make the change :)

File: 5.0/en/0x17-V9-Communications.md (hunk 1 above)
Comment (tghosth) [reply 5.0/en/0x17-V9-Communications.md:54]: Please make the change :)

**Pull request merged**